src/components/SpaceProposalVotesModal.vue

```diff
@@ -1,44 +1,62 @@
 <script setup lang="ts">
+import { clone } from '@snapshot-labs/snapshot.js/src/utils';
 import { useIntersectionObserver } from '@vueuse/core';
-import {
-  ExtendedSpace,
-  Proposal,
-  Vote,
-  SpaceStrategy
-} from '@/helpers/interfaces';
+import { watchDebounced } from '@vueuse/core';
+import { ExtendedSpace, Proposal } from '@/helpers/interfaces';
+
+const VOTES_FILTERS_DEFAULT = {
+  orderDirection: 'desc',
+  onlyWithReason: false
+};
 
 const props = defineProps<{
   space: ExtendedSpace;
   proposal: Proposal;
-  strategies: SpaceStrategy[];
-  userVote: Vote | null;
   open: boolean;
 }>();
 
 defineEmits(['close']);
 
-const votesEndEl = ref<HTMLElement | null>(null);
-const votesQuery = ref('');
-
 const {
+  votes,
+  loadingVotes,
+  loadingMoreVotes,
+  profiles,
   loadVotes,
-  sortedVotes,
-  searchVote,
-  loadMore,
-  loadingMore,
-  loadedVotes,
   loadMoreVotes,
-  profiles,
-  clearVotes,
-  noVotesFound,
-  searchAddress
-} = useProposalVotes(props.proposal, 20, props.userVote, votesQuery);
+  loadSingleVote
+} = useProposalVotes(props.proposal, 20);
+
+const votesEndEl = ref<HTMLElement | null>(null);
+const filterOptions = ref(clone(VOTES_FILTERS_DEFAULT));
+const searchInput = ref('');
+
+const filters = computed(() => {
+  return {
+    orderDirection: filterOptions?.value?.orderDirection || 'desc',
+    onlyWithReason: filterOptions?.value?.onlyWithReason || false
```

Comment:
```suggestion
    orderDirection: filterOptions.value.orderDirection,
    onlyWithReason: filterOptions.value.onlyWithReason
```

`filterOptions` should never be undefined, since instantiated with default values.